index.js

```diff
@@ -54,9 +37,18 @@ module.exports = function staticCache(dir, options, files) {
   }
 
   return function* staticCache(next) {
-    var file = files[safeDecodeURIComponent(path.normalize(this.path))]
-    if (!file)
```

Comment:
```
if (!opts.buffer) {
  // get file from file system directly
} else {
  // keep the old logic
}
```

i think this will be better, in devlop environment, set `options.buffer=false`, in production enviroment, set `options.buffer=true` to cache all the files.